### DISCUSSION TITLE
Lesson 5 Video 7

### DISCUSSION THREAD

**Discussion by cupidchan** [OpenMined/courses#182] [Course: Our Privacy Opportunity] [answered]
@iamtrask gives an example that if both Hospital A and B having an epsilon of 10, then a Patient in both hospitals will leak an epsilon of 20. However, in order to fulfill this, the prerequisite here is I know this Patient is included in these 2 Hospitals. But by the Output Privacy, this should have been already protected. Therefore, unless I can know from another channel that this Patient is actually in the dataset of these 2 Hospitals, it will still be pretty hard, if not impossible, to deduce more insight of that particular Patient with 20 epsilon. Is this understanding correct?

**Comment by iamtrask** [comment, marked as answer]
Hi Cupid - very close. As it turns out that epsilon leakage also includes the probability that you would be able to determine this. Meaning that a major part of leakage is the ability to perform a "linkage attack" and determine whether a person in one dataset is the same as a person in another.

And while of course this might seem unlikely for two randomly chosen hospitals - two hospitals in the same geography is certainly more likely - and of course if you're not looking for a particular person but are instead looking for any kind of overlap (anyone you could de-anonymize) then pretty much any two hospitals near enough to each other are going to have significant patient overlap.

Put another way - output privacy is only a 100% guarantee if there is 0-epsilon leakage.
